### DISCUSSION TITLE
Glad to see this is being worked on again - any plans for Chrome addon support

### DISCUSSION THREAD

**Discussion by cleverestx** [go-shiori/shiori-web-ext#52] [Q&A]
There is an old one from a few years ago, but it doesn't actually "send" when clicked like the newer firefox one does. Would love to see this work with Chrome. Thank you.

**Comment by fmartingr** [comment]
Hey @cleverestx.

I believe the extension present in the chrome web store is not ours, so I doubt it'll get updated.

You can use this extension today by:
- Download the [latest release zip file](https://github.com/go-shiori/shiori-web-ext/releases/tag/v0.8.7)
- Unzip it to a folder
- Under Chrome, go to Extensions page
- Enable "Developer mode"
- Click on "Load unpacked extension"
- Browse where you unzipped the file you downloaded

That should get you the latest extension.

I know is less than optimal and requires manual intervention, but until we complete the API changes I don't have much time to invest on the extension so I'm only doing maintenance work to keep it up-to-date with the latest API and attending PRs.
